### PR TITLE
1.13 Support

### PR DESCRIPTION
**Pull request by Redned235**

### CODE REVIEW

**Comment by Europia79** [comment]
https://github.com/BattlePlugins/BattleTracker/pull/22/commits/a7929a325f98e45789ac14cfc6acce25afa22028

https://bukkit.gamepedia.com/Plugin_YAML

Node `api-version` doesn't exist in the `plugin.yml`

In theory, we should be able to use older versions of Bukkit.jar and still be compatible with future versions of Minecraft because Bukkit is the interface & CraftBukkit.jar is the implementation that changes with each new version of Minecraft.

In practice, however, the devs will sometimes make breaking changes to Bukkit... like in the case of Bukkit.getOnlinePlayers().  This change occured for 1.7.10...

That means that if we had a dependency for Bukkit v1.2.5 then we would be compatible with Minecraft 1.2.5 to 1.7.9

If we had a dependency for Bukkit v1.7.10 then we would be compatible with Minecraft v1.7.10 to 1.13 & beyond into the future... Until they made another breaking change.

Since we NEVER invoke Bukkit.getOnlinePlayers(), then Bukkit v1.2.5 would theoretically make us compatible with all versions of Minecraft, including 1.13 (as long as we only used the common interfaces that didn't change).

Basically, for Bukkit.jar dependency, we always want to use the OLDEST version that we can get away with using.

If we have to depend on Craftbukkit.jar, then that's when we use the NEWEST version.
